docs/how-to-auto-deploy-on-huawei-cloud.md

```diff
@@ -0,0 +1,40 @@
+# Auto deploy microservices on Huawei Cloud
+## Background
+In this workshop, we have 4 services(manager, doorman, beekeeper, worker) and we use [zipkin](http://zipkin.io/) service to trace our handler chain. Hence, there should be 5 applications in this cluster as shown in the following figure(fig.1).
+![fig.1 orchestration architecture](images/cluster_info.png)  
+fig.1 orchestration architecture
+
+## Import orchestration template
+To import the orchestration template, you need the following steps:
+1. Visit [Service Stage's doodle page](https://servicestage.hwclouds.com/servicestage/#/stage/doodle/new/topology).
+2. Click the *Import* button shown in fig.2. 
+3. Click *Select a local template* shown in fig.2.
+4. Select the predefined template file `scripts/workshop.tar.gz`.
+5. Enter the template name, e.g. workshop, then click *Import*.
+![fig.2 import template](images/import_template.png)  
+fig.2 steps to import template  
+Now you should be able to see the orchestration architecture similar to the architecture in fig.1.
+
+## Auto Deploy
+Before you deploy your cluster, you may need to update some properties.
+1. Update cluster name. You can skip this step if your cluster name is workshop.  
+As shown in fig.3, first you need to double click the *cluster-resource* button, then choose the name
+of your cluster.
+![fig.3 update cluster name](images/update_cluster_name.png)  
+fig.3 update cluster name  
+2. Update images' url. If you did not build and publish your images to the image warehouse, you can skip this step and use the public images we provide in the image warehouse.  
+(1) Visit [Huawei's Image Warehouse](https://servicestage.hwclouds.com/servicestage/#/stage/softRepository/mirrorCenter/myMirrorPack), then choose the *workshop\_manager* image. Click on the *copy address* button as shown in fig.4.
```

Comment:
The website calls itself image warehouse, I just use what the website uses in case of users misunderstand or misuse.